app/emu_svc.py

```diff
@@ -47,6 +47,26 @@ async def populate_data_directory(self, path_yaml=None):
         errors_output = f' and ran into {errors} errors' if errors else ''
         self.log.debug(f'Ingested {ingested} abilities (out of {total}) from emu plugin{errors_output}')
 
+    async def decrypt_payloads(self):
+        path_crypt_script = os.path.join(self.repo_dir, '*', 'Resources', 'utilities', 'crypt_executables.py')
+        for crypt_script in glob.iglob(path_crypt_script):
+            plan_path = crypt_script[:crypt_script.rindex('Resources') + len('Resources')]
+            self.log.debug('attempting to decrypt plan payloads using %s with the password "malware"' % crypt_script)
+            process = Popen(['python3', crypt_script, '-i', plan_path, '-p', 'malware', '--decrypt'], stdout=PIPE)
```

Comment:
You can replace `python3` with `sys.executable` to refer to the same Python interpreter (especially useful if using virtualenv).

```suggestion
            process = Popen([sys.executable, crypt_script, '-i', plan_path, '-p', 'malware', '--decrypt'], stdout=PIPE)
```

and `import sys` at the top.